meetings/2016-07-18/minutes.md

```diff
@@ -0,0 +1,139 @@
+# Node Post-Mortem WG Meeting Notes - July 18 2016
+
+* Recording here: http://www.youtube.com/watch?v=HIh7kQUifuU
+* Issue link: https://github.com/nodejs/post-mortem/issues/31
+
+# Present
+
+* Yunong Xiao @yunongx
+* Dave Pacheco @dapsays
+* Joshua M. Clulow @jmclulow
+* Michael Dawson  @mhdawson
+* Richard Chamberlain @rnchamberlain
+* Ali Sheik @ofrobots
+* Luca
+
```

Comment:
Ali and Luca do not seem to have made it to this meeting.

Also, just a personal nit: on GitHub, I assume @ -mentinos are for GitHub- not twitter. (a few above are not the same between the 2 platforms)